### PR TITLE
Adds callout that distributed table engine that it is not supported in Cloud.

### PR DESCRIPTION
**Pull request by johnnymatthews**
### Changelog category (leave one):

- Documentation (changelog entry is not required)

### Changelog entry (a user-readable short description of the changes that goes to CHANGELOG.md):

- Adds callout that this is only for self-hosted users, not cloud users.

### Documentation entry for user-facing changes

- [x] Documentation is written (mandatory for new features)

### CODE REVIEW

**Comment by robot-clickhouse-ci-1** [comment]
*This is an automated comment for commit c6ca43b3418de24069c2544b0f0abe33b0147694 with description of existing statuses. It's updated for the latest CI running*

[❌ Click here](https://s3.amazonaws.com/clickhouse-test-reports/57488/c6ca43b3418de24069c2544b0f0abe33b0147694/ci_running.html) to open a full report in a separate page

<details><summary>Successful checks</summary>
<table>
<thead><tr><th>Check name</th><th>Description</th><th>Status</th></tr></thead>
<tbody>
<tr><td>Mergeable Check</td><td>Checks if all other necessary checks are successful</td><td><a href="https://github.com/ClickHouse/ClickHouse/actions/runs/7093148562">✅ success</a></td></tr>
<tbody>
</table>
</details>
<table>
<thead><tr><th>Check name</th><th>Description</th><th>Status</th></tr></thead>
<tbody>
<tr><td>CI running</td><td>A meta-check that indicates the running CI. Normally, it's in <b>success</b> or <b>pending</b> state. The failed status indicates some problems with the PR</td><td><a href="https://s3.amazonaws.com/clickhouse-test-reports/57488/c6ca43b3418de24069c2544b0f0abe33b0147694/ci_running.html">❌ failure</a></td></tr>
<tbody>
</table>

**Comment by alexey-milovidov** [comment]
The Distributed table engine is supported but has to be created with the `remote` or `remoteSecure` table function:

```
CREATE TABLE ... AS remoteSecure('addresses', database, table, user, password)
```

**Comment by melvynator** [comment]
@johnnymatthews Given that Distributed table are supported in the cloud but not the syntax `Distributed(...)` Instead of adding the generic "This feature is not available in the cloud," we should instead say something like:

:::warning
To create a distributed table engine in the cloud, you can use [remoteSecure and remote](https://clickhouse.com/docs/en/sql-reference/table-functions/remote) table functions. The Distributed(...) syntax cannot be used in the cloud. 
:::

File: docs/en/engines/table-engines/special/distributed.md [hunk 1]
```diff
@@ -1,13 +1,16 @@
...
+slug: /en/engines/table-engines/special/distributed
 ---
 
 # Distributed Table Engine
 
-Tables with Distributed engine do not store any data of their own, but allow distributed query processing on multiple servers.
-Reading is automatically parallelized. During a read, the table indexes on remote servers are used, if there are any.
+:::warning
+To create a distributed table engine in the cloud, you can use the [remote and remoteSecure](../../../sql-reference/table-functions/remote) table functions. The `Distributed(...)` syntax cannot be used in ClickHouse Cloud.
+:::
```
Comment (johnnymatthews) [inline docs/en/engines/table-engines/special/distributed.md:11]: How's this @melvynator?

Comment (melvynator) [reply docs/en/engines/table-engines/special/distributed.md:11]: It looks better.

**Review by melvynator — COMMENTED** [review]
LGTM

**Pull request merged**